Source/TextToSpeechV1/Models/CustomVoiceUpdate.swift

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright IBM Corporation 2016
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+
+public struct CustomVoiceUpdate: JSONEncodable {
+    
+    /// The new name for the custom voice model.
+    public let name: String?
```

Comment:
Since these properties are only used for serialization then the three properties should be set to private.